src/main/resources/avro/rnaquantificationmethods.avdl

```diff
@@ -1,207 +1,146 @@
-'SearchRnaQuantificationRequest' as the post body and will return a JSON
-version of 'SearchRnaQuantificationResponse'.
-*/
-SearchRnaQuantificationResponse searchRnaQuantification(
-    /**
-    This request maps to the body of 'POST /rnaquantification/search'
-    as JSON.
-    */
-    SearchRnaQuantificationRequest request) throws GAException;
-
-/**************** /rnaquantification/{id} *******************/
-/**
-Gets a `RnaQuantification` by ID.
-`GET /rnaquantification/{id}` will return a JSON version of `RnaQuantification`.
-*/
-org.ga4gh.models.RnaQuantification getRnaQuantification(
-  /**
-  The ID of the `RnaQuantification`.
-  */
-  string id) throws GAException;
-
-/****************  /expressionlevel/search  *******************/
-/**
-This request maps to the body of 'POST /expressionlevel/search'
-as JSON.
-*/
-record SearchExpressionLevelRequest {
-  /**
-  If present, return matching Expression Level record.
-  */
-  union { null, string } expressionLevelId = null;
-
-  /**
-  If present return only ExpressionLevel records which belong to this set.
-  */
-  union { null, string } quantificationGroupId = null;
-
-  /**
-  The rnaQuantification to restrict search to.
-  */
-  string rnaQuantificationId;
-
-  /**
-  Only return ExpressionLevel records with expressions exceeding
-  this value.  (Defaults to 0.0)
-  */
-  union { float, null } threshold = 0.0;
-
-
-  /**
-  Specifies the maximum number of results to return in a single page.
-  If unspecified, a system default will be used.
-  */
-  union { null, int } pageSize = null;
-
-  /**
-  The continuation token, which is used to page through large result sets.
-  To get the next page of results, set this parameter to the value of
-  'nextPageToken' from the previous response.
-  */
-union { null, string } pageToken = null;
+// This is the response from 'POST /rnaquantification/search' expressed as JSON.
+message SearchRnaQuantificationResponse {
+  // The list of matching quantifications.
+  repeated RnaQuantification rna_quantification = 1;
+
+  // The continuation token, which is used to page through large result sets.
+  // To get the next page of results, set this parameter to the value of
+  // 'nextPageToken' from the previous response.
+  string next_page_token = 2;
 }
 
-/**
-This is the response from 'POST /expressionlevel/search' expressed as JSON.
-*/
-record SearchExpressionLevelResponse {
-  /** The list of matching quantifications. */
-  array<org.ga4gh.models.ExpressionLevel> expressionLevel = [];
-
-  /**
-  The continuation token, which is used to page through large result sets.
-  To get the next page of results, set this parameter to the value of
-  'nextPageToken' from the previous response.
-  */
-  union { null, string } nextPageToken = null;
+// This request maps to the URL `GET /rnaquantification/{rna_quantification_id}`.
+
+message GetRnaQuantificationRequest {
+  // The ID of the `RnaQuantification`.
+  string rna_quantification_id = 1;
+
```

Comment:
}

These issues should be caught by compilation via mvn once you've rebased onto the protobuf topic branch.